package.json

```diff
@@ -25,6 +25,8 @@
     "lint:js:fix": "eslint . --fix",
     "release": "release-it",
     "start": "ember serve",
+    "start:env": "dotenvx run -- npm run start:proxy",
```

Comment:
This looks a bit strange, but without a second script the env variables aren't yet available to pass into the arguments for ember serve, and we want to reuse the METIS_BACKEND_URL for the proxy value.